release_notes/2.4.13.md

```diff
@@ -1,6 +0,0 @@
-**MS Graph for Office 365 Release Notes - Published by Splunk February 03, 2022**
-
-
-**Version 2.4.13 - Released February 03, 2022**
```

Comment:
Do we need to keep this part onto 2.4.14 with modified date?

Reply:
Nope. We have stopped keeping headers going forward.